build.sh

```diff
@@ -37,6 +37,10 @@ apt-key adv --recv-keys --keyserver keyserver.ubuntu.com F6ECB3762474EDA9D21B702
 # This can be removed when our Debian container has a version containing this fix
 patch -d /usr/lib/live/build/ < 314-follow-symlinks-when-measuring-size-of-efi-files.patch
 
+# TODO: Remove this once debootstrap has a script to build mantic images in our container:
+# https://salsa.debian.org/installer-team/debootstrap/blob/master/debian/changelog
```

Comment:
That's not true anymore for deboostrap > 1.0.132 as per the changelog in Ubuntu:
> Update debootstrap to automatically use the gutsy script by default
      if the target is an Ubuntu release known to distro-info, eliminating
      the need to upload debootstrap for each new Ubuntu release.

Reply:
Well, the build failed because of this:
https://github.com/elementary/os/actions/runs/6221172831/job/16882628733#step:4:645

We don't use the Ubuntu version of `debootstrap` as this is a Debian container, so maybe that's an Ubuntu specific change that didn't make it upstream yet.

Reply:
And, `1.0.128` is the version that gets installed during the build, so it's not new enough anyway.

Reply:
Your change is good, it's only the commend that I would amend